### PR TITLE
Removal of Flask and backend at all (resolves #54)

### LINKED ISSUE #54: Get rid of flask and python

### PR DESCRIPTION
**Pull request by magul**
@gruber86 please review.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,7 @@
-language: python
-python:
-  - "3.4"
+language: node_js
+node_js:
+  - "5.0.0"
```
Comment (gruberrro) [inline .travis.yml:3]: Shpuldn't it be LTS version?

File: src/assets/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!doctype html>
```
Comment (gruberrro) [inline src/assets/index.html:1]: Just wonder, why .index file is inside assets?

**Comment by gruberrro** [comment]
Two doubts, looks fine :+1: 

Edit: You can update README.md as well.

File: .travis.yml (hunk 1 above)
Comment (magul) [inline .travis.yml:3]: Actually we haven't thought about it a lot. The main reason of using 5.0.0 here is that we use `jquery` and `jquery-ui` and there are some side effects (that depends on order of imports) that it doesn't work on older versions of Node. There's a plan to introduce native (https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) and after that I believe we will be able to downgrade node to `0.12` or something like that - then we will test it on multiple versions of it.

BTW If You want, You can check in which this engine work feel free to do this (and add it to `.travis.yml`).

File: src/assets/index.html (hunk 2 above)
Comment (magul) [inline src/assets/index.html:1]: All files that need to be places in root of `/dist` are in `assets` - we can move them anywhere, but if we will place them in main `/src` directory there will be some more logic to be written when we will want to move them to `/dist` directory.

**Comment by magul** [comment]
`README.md` have been updated?

**Comment by gruberrro** [comment]
Looks great!

**Pull request merged**